presto-mysql/pom.xml

```diff
@@ -141,14 +141,8 @@
         </dependency>
 
         <dependency>
-            <groupId>io.airlift</groupId>
-            <artifactId>testing-mysql-server</artifactId>
```

Comment:
@electrum are we going to archive this project now?

Reply:
We still have other usages in Presto, but long term, I think it makes sense to archive it, as we would no longer be maintaining it.

Reply:
I know. One more, i belive.

Reply:
I know. One more, i believe. (https://github.com/prestosql/presto/pull/2410 is related)